meta/parse.pl

```diff
@@ -2549,6 +2549,7 @@ sub ProcessStructValueType
     return "SAI_ATTR_VALUE_TYPE_MAC"            if $type eq "sai_mac_t";
     return "SAI_ATTR_VALUE_TYPE_IP_ADDRESS"     if $type eq "sai_ip_address_t";
     return "SAI_ATTR_VALUE_TYPE_IP_PREFIX"      if $type eq "sai_ip_prefix_t";
+    return "SAI_ATTR_VALUE_TYPE_ACL_CHAIN_LIST" if $type eq "sai_acl_chain_list_t";
```

Comment:
please remove, why this was added here? this struct value should not be LIST, this is probably leftover since you removed acl_chain_list from acl_capability

Reply:
https://github.com/opencomputeproject/SAI/pull/1815/commits/f56ef7b31276c8228817804428421cbf2870c1db